.github/workflows/build.yaml

```diff
@@ -9,29 +9,20 @@ concurrency:
     cancel-in-progress: true
 
 jobs:
-    build-and-publish:
-        name: Build and Push
+    build:
+        name: Build
         runs-on: self-hosted-hoprnet-small
-
+        strategy:
+            matrix:
+                node-version: [20.x, 21.x, 22.x]
```

Comment:
only v20 and v22 should be checked as of now. see https://nodejs.org for current LTS and stable versions

Reply:
Thanks, already removed in all repositories